### ISSUE TITLE
vsphere plugin: Unable to find metric name for id 608

### ISSUE THREAD

**Issue by danielarkins** [influxdata/telegraf#12716]
### Relevant telegraf.conf

```toml
[[inputs.vsphere]]
  interval = "60s"											# default: 180s
  vcenters = [ 	
				"https://vcXXX/sdk",
				"https://vcXXX/sdk"
			 ]
			
  username = "xxx"
  password = "xxx"

  insecure_skip_verify = true

  # Exclude all historical metrics
  datastore_metric_exclude = ["*"]
  cluster_metric_exclude = ["*"]
  datacenter_metric_exclude = ["*"]

  collect_concurrency = 5									# default: 5
  discover_concurrency = 5									# default: 5
```

### Logs from Telegraf

```text
[inputs.vsphere] Unable to find metric name for id 608. Skipping!
```

### System info

Telegraf 1.25.2-67c08c51 on Windows

### Steps to reproduce

1. Apply config
2. execute telegraf
3.
...

### Expected behavior

metrics ingested

### Actual behavior

metrics not ingested

**Comment by danielarkins** [comment]
havent gotten it working, i downgraded to v 1.18.2 where this issue claims to have been resolved but still persists

**Comment by powersj** [comment]
> havent gotten it working

ok, then I am inclined to assume that this ID does not exist unless @prydin knows more about this particular ID. 

You didn't mention which version of vsphere you were on either.

**Comment by danielarkins** [comment]
vsphere 7.3
i believe the id refers to a vm in my estate

**Comment by danielarkins** [comment]
this was an issue reported as resolved in 1.18.2
https://github.com/influxdata/telegraf/pull/8505

**Comment by prydin** [comment]
I've seen this occasionally. It means that there's a metric ID in the vCenter database for which there's no name. I believe this is due to corruption of the vCenter database. It's one of those "This should never happen" situations.

Are you missing any metrics you care about? If not, I would write this up to some junk sitting around in the vCenter database.
